hyftest.js

```diff
@@ -0,0 +1,60 @@
+// 1  Programming Basics
+const movies = ["The Shawshank Redemption",
+"The Godfather",
+"The Dark Knight",
+"12 Angry Men",
+"The Lord Of The Rings",
+"Pulp Fiction",
+"The Good, the Bad and the Ugly",
+"Forrest Gump"];
+
+//1. Log out each movie title;
+console.log(movies)
```

Comment:
As the exercise stated, we expected you to list these titles out **using a `for` loop**. Whilst the solution does result in all titles showing up on the screen, the `[]` characters were not needed.